_63C/rps.md

```diff
@@ -73,18 +73,20 @@ This document covers two main product families that enable federated identity tr
 
 ### C.3.2.2 OpenID Connect {#s-c-3-2-2}
 
 Different OAuth grant types or "flows" are appropriate for different kinds of applications at different FALs.
 
 The authorization code flow is a back-channel presentation mechanism and ought to be used whenever possible, particularly for web server, native, or mobile applications. It is the most common and most secure way to implement OAuth, the underlying protocol of OpenID Connect. It can accommodate all three FALs depending on the exact configuration of the application. The authorization code flow makes use of back channel assertion presentation, which reduces the attack surface of the RP significantly by sending the assertion directly from the IdP to the RP without an intermediary party touching it. The RP ought to authenticate itself when presenting the authorization code to the IdP.
 
-If the RP is a native or mobile application, it can use the [PKCE extension](https://tools.ietf.org/html/rfc7636) or [dynamic client registration](https://tools.ietf.org/html/rfc7591) to ensure that different copies of the client software can't impersonate each other at the IdP. The [best current practices for OAuth 2 mobile applications](https://tools.ietf.org/html/rfc8252) specification provides additional guidance.
+If the RP is a native or mobile application, it can use the [PKCE extension](https://tools.ietf.org/html/rfc7636) or [dynamic client registration](https://tools.ietf.org/html/rfc7591) to ensure that different copies of the client software can not impersonate each other at the IdP. The [best current practices for OAuth 2 mobile applications](https://tools.ietf.org/html/rfc8252) specification provides additional guidance.
 
-In browser applications, sometimes known as Single Page Applications (SPAs) provide their own challenges. The implicit grant type is a front-channel presentation mechanism and is applicable for applications which are implemented entirely in front-end code and have to capability to store secrets outside of the subscriber's web browser. However, best current practice is to use PKCE and the authorization code grant type for an SPA, or to use the OIDC hybrid flows to protect information. These best practices are [being written up by the OAuth working group](https://tools.ietf.org/html/draft-ietf-oauth-browser-based-apps). 
+In-browser applications, sometimes known as Single Page Applications (SPAs), are particularly challenging. The implicit grant type is a front-channel presentation mechanism and is applicable for applications which are implemented entirely in front-end code and have the capability to store secrets outside of the subscriber's web browser. However, best current practice is to use PKCE and the authorization code grant type for an SPA, or to use the OIDC hybrid flows to protect information. Best practices are currently under development by the OAuth working group in [draft-ietf-oauth-browser-based-apps](https://tools.ietf.org/html/draft-ietf-oauth-browser-based-apps). 
```

Comment:
I did not delete the reference to the BCP for SPA's, but I did reformat it into a reference-style link instead.